auth/entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -e
 export CACHE_PORT="${CACHE_PORT:-6379}"
 export MESSAGE_BUS_PORT="${MESSAGE_BUS_PORT:-6379}"
 
-wait-for-it --timeout=10 "$CACHE_HOST:$CACHE_PORT"
-wait-for-it --timeout=10 "$MESSAGE_BUS_HOST:$MESSAGE_BUS_PORT"
+source ./wait-for.sh --timeout=10 "$CACHE_HOST:$CACHE_PORT"
```

Comment:
why do you source the script?

Reply:
Otherwise the script could not be executed.
The other option is to modify it using `chmod +x ./wait-for.sh`.

Reply:
Yes, this is the way. Do it on your local machine and add the file in git again. Then it is executable everywhere